test/ng/supportSpec.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+describe('support test results', function() {
+  var expected, version, testName;
+  var userAgent = window.navigator.userAgent;
+
+  // Support: iOS 8 only
+  if (/iPhone OS 10_1\d(?:_\d+)? /.test(userAgent)) {
+    // iOS 8 official simulators have broken user agent (containing something like `iPhone OS 10_12_5`,
+    // i.e. the macOS version in place of the iOS version so they'd fall into an incorrect bucket.
+    // Fix the user agent there.
+    // NOTE: Make sure the above check doesn't catch the real iOS 10!
+    userAgent = userAgent.replace(/iPhone OS 10(?:_\d)+/, 'iPhone OS 8_1');
+  }
+
+  if (/edge\//i.test(userAgent)) {
```

Comment:
That would have to be `lowerCasedUserAgent` to be precise. And maybe we'll need case sensitivity one day?